content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -16,6 +16,11 @@ customers, and adds an elegant level of professionalism to a company.
 .. seealso::
    `Odoo Quick Tips - Create a PDF quote [video] <https://www.youtube.com/watch?v=tQNydBZt-VI>`_
 
+.. note::
+   It is recommended to edit PDF forms with Adobe software, or similar PDF editing platforms. The
+   form fields on the header and footer PDF templates are necessary, in order to get dynamic values
+   with Odoo.
```

Comment:
Adobe is the only one we know works for sure right now, so let's keep it specific here and we can update later as needed.
```suggestion
   It is recommended to edit PDF forms with Adobe software. The
   form fields on the header and footer PDF templates are necessary to get dynamic values
   with Odoo.
```